torch/onnx/symbolic_opset10.py

```diff
@@ -322,6 +322,38 @@ def isfinite(g, input):
     return __not_(g, __or_(g, inf_node, nan_node))
 
 
+@parse_args("v", "f", "f", "f")
+def nan_to_num(g, input, nan, posinf, neginf):
+    from torch.onnx.symbolic_opset9 import isnan, lt, gt, logical_and
+
+    # Cannot create a int type tensor with inf/nan values, so we simply
+    # return the original tensor
+    if not sym_help._is_fp(input):
+        return input
+    if nan is None:
+        nan = 0.0
+    nan_cond = isnan(g, input)
+    nan_result = g.op("Where", nan_cond,
+                      g.op("Constant", value_t=torch.FloatTensor([nan])), input)
```

Comment:
rather than use FloatTensor, I think we should preserve input's dtype here.
Same for posinf and neginf cases below.